*** Pre-changes	Tue Oct 22 21:34:18 2024
--- Post-changes	Tue Oct 22 21:36:31 2024
***************
*** 18,28 ****
  This includes holding Basic Training {BT} ECATs for new recruits and those transferring in.
  However, as you prove yourself and rank up to 8iC Officer and higher,
  you will gain new responsibilities, such as visiting foreign agencies and hosting events in HQ.
  
  The MI branch is headed by the Director General [4iC]: NoahC500.
! The Deputy Director General [5iC]: GetWild, assists the Director General in the running of the branch,
  and, naturally, deputises when the role is vacant or the Dir. Gen. is on LOA.
! The Personnel Director [6iC]: -nekocho-, focuses on the wellbeing of MI personnel,
  as well as ensuring they complete their assigned duties.
  This makes up MI Senior Leadership.
  The leadership team also includes the Training Strategist [7iC],
--- 18,30 ----
  This includes holding Basic Training {BT} ECATs for new recruits and those transferring in.
  However, as you prove yourself and rank up to 8iC Officer and higher,
  you will gain new responsibilities, such as visiting foreign agencies and hosting events in HQ.
+ All members of MI are expected to both give training sessions and fill stations in HQ.
  
  The MI branch is headed by the Director General [4iC]: NoahC500.
! The Deputy Director General [5iC], assists the Director General in the running of the branch,
  and, naturally, deputises when the role is vacant or the Dir. Gen. is on LOA.
! When the role is vacant, too, the Home Secretary will provide additional aid to the branch,
! The Personnel Director [6iC] focuses on the wellbeing of MI personnel,
  as well as ensuring they complete their assigned duties.
  This makes up MI Senior Leadership.
  The leadership team also includes the Training Strategist [7iC],
***************
*** 32,51 ****
  
  As those in MI often interact with the public,
  through their hosting of events, and during training sessions,
! it’s important for MI personnel to be able to show decorum and be able to be polite, etc..
! One of the main goals of MI is to keep good ’Netgain’.
! Which, to put simply, is the difference in the number of employees since last week.
  Any questions so far?
  Answer any questions
  
! All MI personnel are able to guard the Royal Family, the Cabinet Office and foreign dignitaries
! at special events, such as balls, and the State Opening of Parliament.
! This is as they are able to join the Royalty and Specialist Protection force (RASP).
! RASP is jointly-headed by the Directors General and Personnel.
  
  Members of the MI branch are able to promote, strike, and dismiss LRs.
  This gives the opportunity to reward Low Ranks for their hard work.
  However, this is only after they have successfully passed the Agency Academy Programme (AAP).
  
  Next, I’ll give you a short quiz to assess what you’ve learnt.
  You need 3/4 to pass. If you are unsure of the correct answer, just say ’pass’.
--- 34,58 ----
  
  As those in MI often interact with the public,
  through their hosting of events, and during training sessions,
! it’s important for MI personnel to be able to show decorum and be able to be polite, etc.
! One of the ways the branch analyses its performance is through measuring net gain of employees.
! Net gain is the difference in the number of employees between two distinct times.
! For example, today and yesterday.
  Any questions so far?
  Answer any questions
  
! Royalty and Specialist Protection (RaSP) is a body within MI which guards
! members of the Cabinet, the Royal Family, and foreign dignitaries during special events.
! Examples of such events include treaty signings, the State Opening of Parliament and balls.
! RaSP is jointly-headed by the Directors General and Personnel and all MI personnel may join.
  
  Members of the MI branch are able to promote, strike, and dismiss LRs.
  This gives the opportunity to reward Low Ranks for their hard work.
  However, this is only after they have successfully passed the Agency Academy Programme (AAP).
+ The AAP is a probationary period for all new MI personnel.
+ Through this period, one must first observe a training session.
+ Then, they must themselves train with a member of MI Leadership present.
+ If they performed the session to a satisfactory standard, they may train on their own.
  
  Next, I’ll give you a short quiz to assess what you’ve learnt.
  You need 3/4 to pass. If you are unsure of the correct answer, just say ’pass’.
***************
*** 59,67 ****
  Ensure not to copy the answer along with the question
  1. Who is the current Director General?
  NoahC500
! 2. What is Netgain?
! Number of people who have joined the UK minus the number of people who have left / Difference since last week
! 3. Name one event that relies upon MI members to guard the Royals and members of the Cabinet.
  Royal balls, the State Opening of Parliament, etc.
  4. What are two actions an MI member can perform on an LR?
  Hire, Promote, Demote, Strike, Fire
--- 66,74 ----
  Ensure not to copy the answer along with the question
  1. Who is the current Director General?
  NoahC500
! 2. What is meant by net gain?
! Difference in number of employees between two distinct times
! 3. Name one event which relies upon MI to guard the Royals and members of the Cabinet.
  Royal balls, the State Opening of Parliament, etc.
  4. What are two actions an MI member can perform on an LR?
  Hire, Promote, Demote, Strike, Fire
