*** ./Pre-changes	Tue Nov 19 16:42:09 2024
--- ./Post-changes	Tue Nov 19 16:42:27 2024
***************
*** 35,44 ****
  The Commons Administration is as follows:
  - The Speaker of the House (SotH) [4iC]: EriksS.
  - The Deputy Speaker of the House [5iC].
! - The Clerk of the House [6iC] (assists in bill-drafting & record-keeping).
  The Official Opposition is as follows:
  - The Leader of the Opposition [5iC]: zanetheraptor.
! - The Deputy Leader of the Opposition [6iC]: Lafi.
  - Opposition MP Shadow Secretaries and Ministers [7iC].
  
  
--- 35,44 ----
  The Commons Administration is as follows:
  - The Speaker of the House (SotH) [4iC]: EriksS.
  - The Deputy Speaker of the House [5iC].
! - The Clerk of the House [6iC] (assists in bill-drafting & record-keeping): PowerJaden04.
  The Official Opposition is as follows:
  - The Leader of the Opposition [5iC]: zanetheraptor.
! - The Deputy Leader of the Opposition [6iC].
  - Opposition MP Shadow Secretaries and Ministers [7iC].
  
  
