*** ./Pre-changes	Tue Nov 19 16:46:56 2024
--- ./Post-changes	Tue Nov 19 16:48:18 2024
***************
*** 22,28 ****
  Employees may be struck by their higher-ups (of at least 7iC); all MI employees may strike LRs.
  To show that one has been struck, 'x' followed by the number of strikes must be appended to the offender's motto.
  Strikes typically last two weeks.
! Those who receive three strikes may be fired on-the-spot; those receiving four must be.
  Are there any questions so far?
  Answer any questions
  
--- 22,28 ----
  Employees may be struck by their higher-ups (of at least 7iC); all MI employees may strike LRs.
  To show that one has been struck, 'x' followed by the number of strikes must be appended to the offender's motto.
  Strikes typically last two weeks.
! Those who receive three strikes may be fired on-the-spot; those who received four strikes must be fired.
  Are there any questions so far?
  Answer any questions
  
***************
*** 48,54 ****
  
  In HabboUK, Prime Minister's terms are called 'Parliaments'.
  This is because Parliament is dissolved (closed) whilst a new PM is being elected.
! General Elections take place every four months (next one being in October).
  The winner of the election becomes Prime Minister, the runner-up, the Leader of the Opposition.
  To stand for election, one must have the support of 6 iCs,
  as well as having been an iC themselves for 30 days.
--- 48,54 ----
  
  In HabboUK, Prime Minister's terms are called 'Parliaments'.
  This is because Parliament is dissolved (closed) whilst a new PM is being elected.
! General Elections take place every four months (next one being in November).
  The winner of the election becomes Prime Minister, the runner-up, the Leader of the Opposition.
  To stand for election, one must have the support of 6 iCs,
  as well as having been an iC themselves for 30 days.
