*** ./Pre-changes	Tue Nov 19 16:39:37 2024
--- ./Post-changes	Tue Nov 19 16:39:54 2024
***************
*** 24,33 ****
  The Deputy Director General [5iC], assists the Director General in the running of the branch,
  and, naturally, deputises when the role is vacant or the Dir. Gen. is on LOA.
  When the role is vacant, too, the Home Secretary will provide additional aid to the branch,
! The Personnel Director [6iC] focuses on the wellbeing of MI personnel,
  as well as ensuring they complete their assigned duties.
  This makes up MI Senior Leadership.
! The leadership team also includes the Training Strategist [7iC],
  who focuses on the development of ECATs and other training materials.
  Are there any questions so far?
  Answer any questions
--- 24,33 ----
  The Deputy Director General [5iC], assists the Director General in the running of the branch,
  and, naturally, deputises when the role is vacant or the Dir. Gen. is on LOA.
  When the role is vacant, too, the Home Secretary will provide additional aid to the branch,
! The Personnel Director [6iC]: iPolarity; focuses on the wellbeing of MI personnel,
  as well as ensuring they complete their assigned duties.
  This makes up MI Senior Leadership.
! The leadership team also includes the Training Strategist [7iC]: Didcot;
  who focuses on the development of ECATs and other training materials.
  Are there any questions so far?
  Answer any questions
