*** Pre-changes	Wed Nov 13 01:39:32 2024
--- Post-changes	Wed Nov 13 01:40:57 2024
***************
*** 38,44 ****
  - The Clerk of the House [6iC] (assists in bill-drafting & record-keeping).
  The Official Opposition is as follows:
  - The Leader of the Opposition [5iC]: zanetheraptor.
! - The Deputy Leader of the Opposition [6iC]: -Joey-:.
  - Opposition MP Shadow Secretaries and Ministers [7iC].
  
  
--- 38,44 ----
  - The Clerk of the House [6iC] (assists in bill-drafting & record-keeping).
  The Official Opposition is as follows:
  - The Leader of the Opposition [5iC]: zanetheraptor.
! - The Deputy Leader of the Opposition [6iC]: Lafi.
  - Opposition MP Shadow Secretaries and Ministers [7iC].
  
  
