*** Pre-changes	Sun Dec  1 16:58:18 2024
--- Post-changes	Sun Dec  1 16:58:31 2024
***************
*** 20,26 ****
  
  The Cabinet Office is led by the Prime Minister [1iC]: xCommissioned and their deputy [2iC]: Fiona_.
  The CO is also made up of the First Secretary [3iC]: Enneagram,
! the Cabinet Secretary [4iC]: hears6, and the Cabinet Minister [5iC].
  As well as the Leader of the House of Commons [5iC]: Josephj827,
  whose role is focused on liaising between the two branches and improving activity.
  The department’s main function is to manage the rest of the Government.
--- 20,26 ----
  
  The Cabinet Office is led by the Prime Minister [1iC]: xCommissioned and their deputy [2iC]: Fiona_.
  The CO is also made up of the First Secretary [3iC]: Enneagram,
! the Cabinet Secretary [4iC]: hears6, and the Cabinet Minister [5iC]: -mindthegap-.
  As well as the Leader of the House of Commons [5iC]: Josephj827,
  whose role is focused on liaising between the two branches and improving activity.
  The department’s main function is to manage the rest of the Government.
