*** Pre-changes	Wed Nov 13 01:46:13 2024
--- Post-changes	Wed Nov 13 01:48:50 2024
***************
*** 10,16 ****
  Are there any questions before we begin?
  Answer any questions
  
- 
  The Executive branch is one of the four main branches of the UK and is run by the Cabinet Office.
  It is made up of various ministries, departments, and offices, all with their own responsibilities.
  To join the branch, one must be appointed to a position in the Executive.
--- 10,15 ----
***************
*** 19,27 ****
  Any questions so far?
  Answer any questions
  
! The Cabinet Office is led by the Prime Minister [1iC]: Seas. and their deputy [2iC]: xCommissioned.
  The CO is also made up of the First Secretary [3iC]: Enneagram,
! the Cabinet Secretary [4iC], and the Cabinet Minister [5iC]: hears6.
  As well as the Leader of the House of Commons [5iC]: Josephj827,
  whose role is focused on liaising between the two branches and improving activity.
  The department’s main function is to manage the rest of the Government.
--- 18,26 ----
  Any questions so far?
  Answer any questions
  
! The Cabinet Office is led by the Prime Minister [1iC]: xCommissioned and their deputy [2iC]: Fiona_.
  The CO is also made up of the First Secretary [3iC]: Enneagram,
! the Cabinet Secretary [4iC]: hears6, and the Cabinet Minister [5iC].
  As well as the Leader of the House of Commons [5iC]: Josephj827,
  whose role is focused on liaising between the two branches and improving activity.
  The department’s main function is to manage the rest of the Government.
***************
*** 47,59 ****
  The Foreign Office also manages the Diplomatic Service Officer (DSO) team.
  FSOs are volunteers who visit other agencies and report back to the Foreign Office.
  8iC MI Officers are de facto Security Officers (SOs); their role is similar to that of DSO.
! The current Foreign Sec. is: KyleGunzkofer8; the Minister is PowerJaden04.
  Are there any questions?
  Answer any questions
  
  The Home Office is responsible for the management of personnel and transfer requests.
  This is as well as aiding in the oversight of the MI branch.
! The current Secretary is: Fiona_; the Minister for the Home Department is kevincool@.
  
  The Ministry of Defence is responsible for ensuring the safety of the UK.
  They deal with all security threats, and control the Ban on Sight (BOS) list published onsite.
--- 46,58 ----
  The Foreign Office also manages the Diplomatic Service Officer (DSO) team.
  FSOs are volunteers who visit other agencies and report back to the Foreign Office.
  8iC MI Officers are de facto Security Officers (SOs); their role is similar to that of DSO.
! The current Foreign Sec. is: TheHarveh; the Minister is PowerJaden04.
  Are there any questions?
  Answer any questions
  
  The Home Office is responsible for the management of personnel and transfer requests.
  This is as well as aiding in the oversight of the MI branch.
! The current Secretary is: KyleGunzkofer8; the Minister for the Home Department is kevincool@.
  
  The Ministry of Defence is responsible for ensuring the safety of the UK.
  They deal with all security threats, and control the Ban on Sight (BOS) list published onsite.
