*** Pre-changes	2024-11-27 23:03:24.431476039 +0000
--- Post-changes	2024-11-27 23:03:32.211393629 +0000
***************
*** 53,59 ****
  Its leadership consists of:
  - The Lord Speaker [4iC] (presides over the House): Apped.
  - The Deputy Lord Speaker [5iC]: Habtaz.
! - The Clerk of the House of Lords [6iC]: supaman981.
  
  
  The rest of the House of Lords consists of:
--- 53,59 ----
  Its leadership consists of:
  - The Lord Speaker [4iC] (presides over the House): Apped.
  - The Deputy Lord Speaker [5iC]: Habtaz.
! - The Clerk of the House of Lords [6iC].
  
  
  The rest of the House of Lords consists of:
