*** Pre-changes	Tue Nov 26 23:44:28 2024
--- Post-changes	Tue Nov 26 23:43:41 2024
***************
*** 29,35 ****
  handling complaints from and about other agencies, and managing the National Punishment (NP) list.
  People on the NP list (and their alts) are banned from being employed in the UK and its allies.
  The leader of the Foreign Office is the Foreign Secretary [5iC]: dz?.
! Their deputy is the Foreign Minister [6iC].
  Are there any questions so far?
  Answer any questions
  
--- 29,35 ----
  handling complaints from and about other agencies, and managing the National Punishment (NP) list.
  People on the NP list (and their alts) are banned from being employed in the UK and its allies.
  The leader of the Foreign Office is the Foreign Secretary [5iC]: dz?.
! Their deputy is the Foreign Minister [6iC]: Coolvimto.
  Are there any questions so far?
  Answer any questions
  
***************
*** 59,61 ****
--- 59,66 ----
  
  
  Correct their answers
+ 
+ 
+ Please let me or another member of MI know if you wish to take any more ECATs.
+ Are there any questions before we finish up?
+ Answer any questions
