*** Pre-changes	Tue Nov 26 23:46:34 2024
--- Post-changes	Tue Nov 26 23:48:06 2024
***************
*** 46,52 ****
  The Foreign Office also manages the Diplomatic Service Officer (DSO) team.
  FSOs are volunteers who visit other agencies and report back to the Foreign Office.
  8iC MI Officers are de facto Security Officers (SOs); their role is similar to that of DSO.
! The current Foreign Sec. is: dz?.
  Are there any questions?
  Answer any questions
  
--- 46,52 ----
  The Foreign Office also manages the Diplomatic Service Officer (DSO) team.
  FSOs are volunteers who visit other agencies and report back to the Foreign Office.
  8iC MI Officers are de facto Security Officers (SOs); their role is similar to that of DSO.
! The current Foreign Sec. is: dz?; their minister is Coolvimto.
  Are there any questions?
  Answer any questions
  
***************
*** 73,79 ****
  the organisation’s Discord server and website.
  It also ensures all UK materials published onsite are up to date and adhere to policy.
  The department also liaises with the DCME on any publications surrounding upcoming events.
! The current web developer is zanetheraptor.
  
  Next, I’ll give you a short quiz to assess what you’ve learnt.
  You need 3/5 to pass. If you are unsure of the correct answer, just say ’pass’.
--- 73,79 ----
  the organisation’s Discord server and website.
  It also ensures all UK materials published onsite are up to date and adhere to policy.
  The department also liaises with the DCME on any publications surrounding upcoming events.
! The current web developers are Owen? and zanetheraptor.
  
  Next, I’ll give you a short quiz to assess what you’ve learnt.
  You need 3/5 to pass. If you are unsure of the correct answer, just say ’pass’.
***************
*** 98,103 ****
  
  Correct their answers
  
! Please let me or a member of MI if you wish to take any more ECATs.
  Are there any questions before we finish up?
  Answer any questions
--- 98,103 ----
  
  Correct their answers
  
! Please let me or another member of MI know if you wish to take any more ECATs.
  Are there any questions before we finish up?
  Answer any questions
