*** ./Pre-changes	Wed Nov 27 00:34:58 2024
--- ./Post-changes	Wed Nov 27 00:35:22 2024
***************
*** 37,44 ****
  - The Deputy Speaker of the House [5iC].
  - The Clerk of the House [6iC] (assists in bill-drafting & record-keeping): PowerJaden04.
  The Official Opposition is as follows:
! - The Leader of the Opposition [5iC]: zanetheraptor.
! - The Deputy Leader of the Opposition [6iC].
  - Opposition MP Shadow Secretaries and Ministers [7iC].
  
  
--- 37,44 ----
  - The Deputy Speaker of the House [5iC].
  - The Clerk of the House [6iC] (assists in bill-drafting & record-keeping): PowerJaden04.
  The Official Opposition is as follows:
! - The Leader of the Opposition [5iC]: UsernameIsInUse.
! - The Deputy Leader of the Opposition [6iC]: Jurbi.
  - Opposition MP Shadow Secretaries and Ministers [7iC].
  
  
***************
*** 139,145 ****
  Correct their answers
  
  
! Please consider joining the Commons if you think it is the branch for you!
! Please let me or a member of MI if you wish to take any more ECATs.
  Are there any questions before we finish up?
  Answer any questions
--- 139,145 ----
  Correct their answers
  
  
! Do consider joining the Commons if you think it is the branch for you!
! Please let me or another member of MI know if you wish to take any more ECATs.
  Are there any questions before we finish up?
  Answer any questions
