*** Pre-changes	2024-10-31 23:39:10.608040468 +0000
--- Post-changes	2024-10-31 23:39:40.201254150 +0000
***************
*** 19,35 ****
  Any questions so far?
  Answer any questions
  
! The Cabinet Office is led by the Prime Minister [1iC]: Apped and their deputy [2iC]: Jush.
! The CO is also made up of the First Secretary [3iC]: Fiona_,
! the Cabinet Secretary [4iC]: hears6, and the Cabinet Minister [5iC].
! As well as the so-called Leader of the House of Commons [5iC]: EpicMorilecime,
  whose role is focused on liaising between the two branches and improving activity.
  The department’s main function is to manage the rest of the Government.
  This is along with advising branch leaders and performing MP duties.
  Any questions so far?
  Answer any questions
  
! His Majesty’s Treasury (HMT) is run by the Chancellor of the Exchequer [5iC];
  as well as their deputy (the Exchequer Secretary [6iC]).
  Its purpose is managing UK finances and paid perks, as well as regulating the VIP badge.
  
--- 19,35 ----
  Any questions so far?
  Answer any questions
  
! The Cabinet Office is led by the Prime Minister [1iC]: Seas. and their deputy [2iC]: xCommissioned.
! The CO is also made up of the First Secretary [3iC]: Enneagram,
! the Cabinet Secretary [4iC], and the Cabinet Minister [5iC]: hears6.
! As well as the Leader of the House of Commons [5iC]: Josephj827,
  whose role is focused on liaising between the two branches and improving activity.
  The department’s main function is to manage the rest of the Government.
  This is along with advising branch leaders and performing MP duties.
  Any questions so far?
  Answer any questions
  
! His Majesty’s Treasury (HMT) is run by the Chancellor of the Exchequer [5iC]: Jush;
  as well as their deputy (the Exchequer Secretary [6iC]).
  Its purpose is managing UK finances and paid perks, as well as regulating the VIP badge.
  
***************
*** 38,44 ****
  Hearings may be held for breach of law,
  or to interpret a piece of legislation where definitions are muddy or unclear.
  It is led by the Justice Secretary [5iC]: dreckly,
! as well as their deputy, the Attorney General [6iC].
  
  The rest of the departments are led by a Secretary of State [5iC], whose deputy is a 6iC Minister.
  
--- 38,44 ----
  Hearings may be held for breach of law,
  or to interpret a piece of legislation where definitions are muddy or unclear.
  It is led by the Justice Secretary [5iC]: dreckly,
! as well as their deputy, the Attorney General [6iC]: SycamoreTree.
  
  The rest of the departments are led by a Secretary of State [5iC], whose deputy is a 6iC Minister.
  
***************
*** 47,64 ****
  The Foreign Office also manages the Diplomatic Service Officer (DSO) team.
  FSOs are volunteers who visit other agencies and report back to the Foreign Office.
  8iC MI Officers are de facto Security Officers (SOs); their role is similar to that of DSO.
! The current Foreign Sec. is: Seas.; the Minister is Enneagram.
  Are there any questions?
  Answer any questions
  
  The Home Office is responsible for the management of personnel and transfer requests.
  This is as well as aiding in the oversight of the MI branch.
! The current Secretary is: KyleGunzkofer8.
  
  The Ministry of Defence is responsible for ensuring the safety of the UK.
  They deal with all security threats, and control the Ban on Sight (BOS) list published onsite.
  They work closely with MI to ensure that the employees have a safe working environment.
! The current Defence Sec. is: xCommissioned, the Minister is: agumobrown.
  Any questions?
  Answer any questions
  
--- 47,64 ----
  The Foreign Office also manages the Diplomatic Service Officer (DSO) team.
  FSOs are volunteers who visit other agencies and report back to the Foreign Office.
  8iC MI Officers are de facto Security Officers (SOs); their role is similar to that of DSO.
! The current Foreign Sec. is: KyleGunzkofer8; the Minister is PowerJaden04.
  Are there any questions?
  Answer any questions
  
  The Home Office is responsible for the management of personnel and transfer requests.
  This is as well as aiding in the oversight of the MI branch.
! The current Secretary is: Fiona_; the Minister for the Home Department is kevincool@.
  
  The Ministry of Defence is responsible for ensuring the safety of the UK.
  They deal with all security threats, and control the Ban on Sight (BOS) list published onsite.
  They work closely with MI to ensure that the employees have a safe working environment.
! The current Defence Sec. is: agumobrown, the Minister is: L.Cornwall.
  Any questions?
  Answer any questions
  
***************
*** 69,75 ****
  Culture Coordinators are responsible for hosting events in HQ.
  8iC+ MI5 agents are ex officio Culture Coordinators.
  To join, please reach out to the Culture Secretary or Minister,
! currently Haree and PowerJaden04 respectively.
  
  The Department for Technology is responsible for the maintenance and improvement of
  the organisation’s Discord server and website.
--- 69,75 ----
  Culture Coordinators are responsible for hosting events in HQ.
  8iC+ MI5 agents are ex officio Culture Coordinators.
  To join, please reach out to the Culture Secretary or Minister,
! currently Haree and Half.Blood respectively.
  
  The Department for Technology is responsible for the maintenance and improvement of
  the organisation’s Discord server and website.
