*** Pre-changes	Wed Nov 13 01:36:52 2024
--- Post-changes	Wed Nov 13 01:37:14 2024
***************
*** 28,34 ****
  This is as well as community engagement (i.e. inter-agency events); crisis management;
  handling complaints from and about other agencies, and managing the National Punishment (NP) list.
  People on the NP list (and their alts) are banned from being employed in the UK and its allies.
! The leader of the Foreign Office is the Foreign Secretary [5iC]: KyleGunzkofer8.
  Their deputy is the Foreign Minister [6iC]: PowerJaden04.
  Are there any questions so far?
  Answer any questions
--- 28,34 ----
  This is as well as community engagement (i.e. inter-agency events); crisis management;
  handling complaints from and about other agencies, and managing the National Punishment (NP) list.
  People on the NP list (and their alts) are banned from being employed in the UK and its allies.
! The leader of the Foreign Office is the Foreign Secretary [5iC]: TheHarveh.
  Their deputy is the Foreign Minister [6iC]: PowerJaden04.
  Are there any questions so far?
  Answer any questions
***************
*** 51,57 ****
  
  
  1. Who is the current Foreign Secretary?
! KyleGunzkofer8
  2. What might a Foreign Service Officer do?
  Any from: Visit other agency, complete events, compile report
  3. Name one responsibility of the Foreign Office.
--- 51,57 ----
  
  
  1. Who is the current Foreign Secretary?
! TheHarveh
  2. What might a Foreign Service Officer do?
  Any from: Visit other agency, complete events, compile report
  3. Name one responsibility of the Foreign Office.
