*** Pre-changes	2024-10-31 22:16:01.848981245 +0000
--- Post-changes	2024-10-31 22:16:23.295552446 +0000
***************
*** 9,20 ****
  Are there any questions before we begin?
  Answer any questions
  
! Throughout your time within the UK, to progress, you will be required to pass a series of ECATs.
! This stands for Education, Certification and Training,
! these sessions will give you the info you need to be an effective employee.
! Again, this is the Basic Training ECAT,
! thusly it will inform you of the basic rules of our establishment,
! as well as its structure and the duties you may need to perform.
  Are there any questions?
  Answer any questions
  
--- 9,21 ----
  Are there any questions before we begin?
  Answer any questions
  
! Throughout your time in the UK, in order to progress, you will need to pass a variety of training sessions.
! These are known as ECATs (Education, Certification and Training),
! and the required ones are published on the HabboUK»com website.
! These sessions will give you the info you need to be an effective employee.
! This session is ECAT {BT} - Basic Training
! and will therefore inform you on the UK’s basic rules and structures,
! as well as any duties you may need to perform.
  Are there any questions?
  Answer any questions
  
***************
*** 39,46 ****
  
  The Prime Minister [1iC] is the head of the UK Government,
  and is directly elected by iC employees every four months.
! The current Prime Minister (PM) is Apped, the Deputy PM [2iC] is Jush.
! 
  Any questions so far?
  Answer any questions
  
--- 40,46 ----
  
  The Prime Minister [1iC] is the head of the UK Government,
  and is directly elected by iC employees every four months.
! The current Prime Minister (PM) is Seas., the Deputy PM [2iC] is xCommissioned.
  Any questions so far?
  Answer any questions
  
***************
*** 94,100 ****
  
  
  1. Who is the current Prime Minister?
! Apped
  2. Who is the current King?
  Terrorization
  3. Name two commands and what they mean.
--- 94,100 ----
  
  
  1. Who is the current Prime Minister?
! Seas.
  2. Who is the current King?
  Terrorization
  3. Name two commands and what they mean.
